loglist/loglist.go

```diff
@@ -168,6 +167,8 @@ func (ll *LogList) FindLogByKey(key []byte) *Log {
 	return nil
 }
 
+var hexDigits = regexp.MustCompile("^[0-9a-fA-F]+$")
```

Comment:
can we use hex.DecodeString instead of a regex?

Reply:
`hex.DecodeString` doesn't allow odd number of digits.

Reply:
How odd.